lib/company/opening.js

```diff
@@ -223,8 +223,63 @@ function deleteOpeningOfCompany(userId, openingId, callback) {
   );
 }
 
+/**
+ * Function to extract all the openings of the company
+ * @param {ObjectId} companyId company Id
+ * @param {ObjectId} openingId opening Id
+ * @param {Function} callback having error and Fetched data
+ */
+function getAllOpenings(companyId, openingId, callback) {
+  JobOpening.find(
+    {
+      company_id: companyId,
+    },
+    {
```

Comment:
This is wrong we dont require `_id` conditions , putting this condition will give only one opening and not all openings.